content/docs/other-guides/multi-user-overview.md

```diff
@@ -4,6 +4,8 @@ description = "Isolation of user-created resources for convenience and organizat
 weight = 50
 +++
 
+{{% stable-status %}}
```

Comment:
This page doesn't seem to be an "application" so it isn't included in the application matrix.  Does there need to be a separate list of "stable components" or anything like that? Maybe that just gets included in release notes.

Reply:
The "multi-user isolation" feature is actually achieved by the profile controller, which is in the table. I've added the words "multi-user isolation" into the table to make this clear. Thanks!